### ISSUE TITLE
PyPI v1.0.6 release .pyx file missing

### ISSUE THREAD

**Issue by ViKingIX** [aresch/rencode#27]
As the title says, rencode.pyx is missing in the PyPI v1.0.6 release. Please repack the PyPI v1.0.6 with the pyx, thanks

**Comment by eibe** [comment]
Yes, _renode.pyx (note the underscore) is missing in the release. The latest [commit ](https://github.com/aresch/rencode/commit/ffc40a7d3fb9f7007eb93abf98949aa752a7ffae#diff-327315918f2280abefdb8e286e15032d) is also missing.

**Comment by jrollins21** [comment]
Until a new release is made, I had success installing with pip using this command:

`pip install -e git+https://github.com/aresch/rencode.git@5c928f14567fabc9efb8bbb8ac5e0eef03c61541#egg=rencode`
